app/screens/browse_channels/search_handler.tsx

```diff
@@ -36,11 +36,22 @@ const MIN_CHANNELS_LOADED = 10;
 const LOAD = 'load';
 const STOP = 'stop';
 
-const filterChannelsByTerm = (channels: Channel[], term: string) => {
-    const lowerCasedTerm = term.toLowerCase();
-    return channels.filter((c) => {
-        return (c.name.toLowerCase().includes(lowerCasedTerm) || c.display_name.toLowerCase().includes(lowerCasedTerm));
-    });
+const filterChannelsByType = (channels: Channel[], joinedChannels: MyChannelModel[], channelType: string) => {
```

Comment:
Not urgent / important but having a `type ChannelType = 'archived' | 'shared' | 'public'` would be great, though I know it'll mean adding it in a lot of places.

Reply:
We have already a ChannelType type that covers open, direct, gm, and private. We could create another "channeltype" type, but we probably should use a different name. Thoughts?